Gemfile

```diff
@@ -41,8 +39,7 @@ gem 'omniauth-rails_csrf_protection'
 
 gem 'simple_form'
 
-gem 'paperclip', '~> 5.2'
-gem 'aws-sdk', '~> 2.3.0'
```

Comment:
We don't need this anymore?

Reply:
I think we do, but only in production.

Reply:
I mis-read the [`kt-paperclip` docs](https://betterment.slack.com/archives/C02FK1484/p1677765605320289). Looked again this morning and they only include the SDK as a dev dependency. I re-added it.